### PR TITLE
Update metrics config to global

### PR DESCRIPTION
**Pull request by Ardagan**
## What was changed

Per-service metrics config is deprecated. Use global config section instead.

## Checklist

1. Closes 

2. How was this tested:

manual `helm template` run

3. Any docs updates needed?

### CODE REVIEW

File: templates/server-configmap.yaml [hunk 1]
```diff
@@ -1,36 +1,37 @@
 {{- if $.Values.server.enabled }}
+{{- range $service := (list "frontend" "history" "matching" "worker") }}
```
Comment (underrun) [inline templates/server-configmap.yaml:2]: is the intention here is to create a different config map with a different `config_template.yaml` for every service?

Comment (jbreiding) [reply templates/server-configmap.yaml:2]: not a different tag but a different tag value?

but that tag value also maps to the env var that directs the service of which service profile to run.

this feels like something that should also be driven from that value.

https://github.com/temporalio/helm-charts/blob/4c7ee04165dc2bfe6dcbb67abbda6c0ac98521e7/templates/server-deployment.yaml#L98

**Review by jbreiding — CHANGES_REQUESTED** [review]
Seems like the variation of tag value should default coming from the service itself.

**Comment by Ardagan** [comment]
I'll close this pr for now. Will update after some more discussion.

**Pull request closed without merging**